types/html-to-pdfmake/index.d.ts

```diff
@@ -11,7 +11,8 @@ declare function htmlToPdfmake(
     html: string,
     options?: Partial<{
         /**
-         * You can overwrite the default styles using defaultStyles
+         * You can overwrite the default styles using defaultStyles.
+         * Please, note that the above default styles are stronger than the ones defined in the style classes.
          */
         defaultStyles: import("pdfmake/interfaces").StyleDictionary;
```

Comment:
I don't think this is correct… `defaultStyles` has nothing to do with `pdfmake`

Reply:
Looking at your sources I think so too. But which style attributes are allowed? Everything html allows?

Reply:
> But which style attributes are allowed? Everything html allows?

Yes, with the camel case (e.g. "fontSize", "marginBottom", …)

Reply:
I'm a little confused. The example `defaultStyles` in your Readme contains `bold: true`, `decoration: "underline"`, and `italics: true`, which are not valid CSS attributes but pdfmake properties.

Reply:
You're right… It's indeed the PDFMake style. Sorry for the confusion!

Reply:
Alright, I think the types are now correct. Thanks for your help!
Could you please approve the PR so the maintainers can see this PR is ready to merge?